File: repos_estudiantes_TARQUI_primera_entrega/LUNA_CRUZ_CAMILA_MILENKA/Ejercicio3/Coche.java
//3.Crea una clase Coche con marca,modelo y velocidad
package Ejercicio3;

public class Coche {
    private String marca;
    private String modelo;
    private int velocidad;


    public Coche(String marca,String modelo,int velocidad){
        this.marca=marca;
        this.modelo=modelo;
        this.velocidad=velocidad;
    }
    public void acelerar(){
        velocidad=velocidad+10;
        System.out.println("------------------------------");
        System.out.println("La velocidad aumenta a: " + velocidad);
    }
    public void frenar(){
        velocidad=velocidad-5;
        System.out.println("------------------------------");
        System.out.println("La velocidad disminuyo a: " + velocidad);
    }
    public void mostrar(){
        System.out.println("Coche: " + marca + " modelo: " + modelo + " -velocidad: " + velocidad + " Km/h");
    }
    public static void main(String[] args) {
        Coche coche1=new Coche ("TOYOTA","HILUX",40);
        Coche coche2=new Coche ("ZUZUKI","TAYRON",85);

        coche1.mostrar();
        coche2.mostrar();
        System.out.println("------------------------------");
        System.out.println("velocidad tras acelerar: ");
        coche1.acelerar();
        coche2.acelerar();
        System.out.println("------------------------------");
        System.out.println("frenar: ");
        coche1.frenar();
        coche2.frenar();


        
    }
    

    
}

File: repos_estudiantes_TARQUI_primera_entrega/QUISPE_MALDONADO_CINTHIA_JAZMIN/INTRODUCCION-POO/EJERCICIO-3/Coche.py
class Coche:
    def __init__(self, marca, modelo, velocidad):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad

    def acelerar(self):
        self.velocidad = self.velocidad + 10
        print("El coche esta acelerando ahora su velocidad es de:", self.velocidad)

    def frenar(self):
        self.velocidad = self.velocidad - 5
        if self.velocidad < 0:
            self.velocidad = 0
        print("El coche esta frenando su velocidad es de:", self.velocidad)

    

c1 = Coche("Toyota", "Corolla", 56)
c2 = Coche("Honda", "Civ", 39)

print("Marca:", c1.marca, ", Modelo:", c1.modelo, ", Velocidad:", c1.velocidad)
c1.acelerar()
c1.frenar()


print("Marca:", c2.marca, ", Modelo:", c2.modelo, ", Velocidad:", c2.velocidad)
c2.acelerar()
c2.frenar()
